src/test/ui/unsized/unsized-fn-arg.rs

```diff
@@ -0,0 +1,6 @@
+// run-rustfix
+#![crate_type="lib"]
+#![allow(unused)]
+
+fn f<T: ?Sized>(t: T) {}
+//~^ ERROR the size for values
```

Comment:
```suggestion
//~^ ERROR the size for values of type `T` cannot be known at compilation time
```
The message is not too long, if it can be written in a way understandable without extra context, then it's better to do that.